docs/content/latest/explore/cluster-management/_index.md

```diff
@@ -13,7 +13,7 @@ menu:
   latest:
```

Comment:
@lizayugabyte pls delete the aliases section in this file. It conflicts with `point-in-time-recovery-ycql.md`

Reply:
Deleted